source/_docs/energy/individual-devices.markdown

```diff
@@ -24,6 +24,9 @@ We recommend the Shelly brand because they have a local API that updates Home As
 - [Shelly 2.5](https://shop.shelly.cloud/shelly-2.5-ce-ul-wifi-smart-home-automation?tracking=A7FsiPIfUWsFpnfKHa8SRyUYLXjr2hPq)
  {% endcomment %}
 
+### Devices with current (W) sensors
```

Comment:
The parent chapter is about hardware options available, I don't understand how this is related?

Reply:
Actually, I also wasn't sure. I guess this should be its own section, on Devices?